src/extension/background-script/connectors/lnd.ts

```diff
@@ -81,7 +83,58 @@ class Lnd implements Connector {
       };
     });
   }
+  async sendPaymentKeySend(
+    args: SendPaymentArgs
+  ): Promise<SendPaymentResponse> {
+    //See: https://gist.github.com/dellagustin/c3793308b75b6b0faf134e64db7dc915
+    const byteArray = new Uint8Array(32);
 
+    const dest_pubkey_hex = args.pubkey;
+    const dest_pubkey_base64 = Buffer.from(dest_pubkey_hex, "hex").toString(
+      "base64"
+    );
+    console.log(dest_pubkey_base64);
+
+    for (let i = 0; i < 32; i++) {
+      const randomNumber = _.random(0, 255);
+      byteArray[i] = randomNumber;
+    }
```

Comment:
we should move this into a utils function: `utils.randomHex(32)` or something. 
@dylancom do you know a nicer way on how to get a random hex string?